packages/react-components/src/Tabs/index.tsx

```diff
@@ -38,38 +56,98 @@ function Tabs ({ basePath, className = '', hidden, isSequence, items }: Props):
     }
   }, [basePath, hidden, items, location]);
 
+  const hasSudo = useMemo(
+    () => !!sudoKey && allAccounts.some((address) => sudoKey.eq(address)),
+    [allAccounts, sudoKey]
+  );
+
+  const groupRef = useRef({
+    accounts: t('Accounts'),
+    developer: t('Developer'),
+    governance: t('Governance'),
+    network: t('Network')
+  });
```

Comment:
Ok, all these changes needs to be reverted. 

- We really don't want to maintain this in two places
- We now have the components fully tied to apps (the medium-term is to move them out)
- Functionally, this approach is confusing as mentioned